src/models.py

```diff
@@ -12,3 +14,12 @@ class User(Base):  # pylint: disable=too-few-public-methods
 
     id = Column(Integer, primary_key=True)  # noqa: A003
     username = Column(String, nullable=False, unique=True)
+    password_hash = Column(String(64), nullable=False)
+
+    def set_password(self, password: str) -> None:
+        """Use this method to set hashed password to User."""
+        self.password_hash = hashlib.sha256(str(password).encode()).hexdigest()
```

Comment:
Seems we don't need to use `str()` here because `password` already has a type `str` accordingly to the type annotation